client/src/apps/code-collabo/styles/app.main.js

```diff
@@ -14,30 +14,39 @@ export default css.global`
     }
 
     #__next {
-        // --- spacing ontop & below page
+        // ---belongs to library: page structure
+        display: block;
+    
+        // ---specific to our app: spacing ontop & below page
         padding: 30px 0;
-        position: relative;
     }
 
     .app__menubar__nav {
-
-    }
-
-    .app__menubar__nav__mobile {
+        // ---belongs to library
         position: fixed;
+
+        // ---belongs to library: this is only needed for when position is fixed
         top: 0;
-        margin: 0 auto;
-        width: calc(100% - 60px);
+        left: 0;
+
+        // ---specific to our app
+        background: var(--color-shade-white);
+        width: 300px;
         height: 100%;
-        background: #e3e3e3;
     }
 
-    .app__menubar__nav__mobile.open {
-      transform: translateX(-100%); /* Bring sidebar in view */
+    .app__menubar__toggle__close {
```

Comment:
Review comment for myself. Maybe we can just only one toggle button, and just switch images instead.
Instead of having 2 of these buttons that do the work of toggling.